### DISCUSSION TITLE
支持桌面版吗

### DISCUSSION THREAD

**Discussion by blue3xp** [NervJS/taro#11649] [Q & A]
### 这个特性解决了什么问题？
可以通过一套代码实现PC和手机的接入

### 这个 API 长什么样？
通过responsive css

**Comment by CaiDingxian** [comment]
小程序也支持媒体查询，只要结合js控制好条件，PC端当然是可以运行的...

**Comment by SyMind** [comment]
你这是需要开发桌面应用么？
